docs/manual/developer_guide.adoc

```diff
@@ -555,8 +555,50 @@ Contributions can be made for rules, checks, remediations or even utilities. The
 
 ==== Rules
 
-...
-// shared/references/cce-rhel-avail.txt
+Rules are input in a simplified XCCDF format, which is basically an XML container.
+Rules are defined as members of a `Group` in a XML file, most likely below the `shared/xccdf` directory.
```

Comment:
Can you make clear when a Rule should be in `shared/xccdf`? I know it is mentioned at end of section 8, but it sounds very vague here.